### PR TITLE
Creating the first test and setting up the tests with TypeScript

### PR DESCRIPTION
**Pull request by Sebasgl23**
By error i made the commit of the exercise 4 and 5 in one pull request, sorry for that.

image

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```
Comment (AlejaGonzalezV) [inline cypress/support/commands.js]: Please add an empty line at the end of the file (EOF)

File: cypress/support/e2e.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
Comment (AlejaGonzalezV) [inline cypress/support/e2e.js]: Please add an empty line at the end of the file (EOF)

File: cypress/support/commands.js (hunk 1 above)
Comment (Sebasgl23) [reply cypress/support/commands.js]: Done

File: cypress/support/e2e.js (hunk 2 above)
Comment (Sebasgl23) [reply cypress/support/e2e.js]: Done

File: cypress/support/e2e.js [hunk 3]
```diff
@@ -17,4 +17,5 @@
 import './commands'
 
 // Alternatively you can use CommonJS syntax:
-// require('./commands')
\ No newline at end of file
+// require('./commands')
+=======
```
Comment (holgiosalos) [inline cypress/support/e2e.js]: remove this line

Comment (Sebasgl23) [reply cypress/support/e2e.js]: Solved

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**